docs/model_control_mode.md

```diff
@@ -0,0 +1,152 @@
+# Model Control Mode
+
+TorchServe now supports model control mode with two settings "none"(default) and "explicit"
+
+## How to set Model Control to explicit
+1. Add `--model-mode-explicit` to command line when running TorchServe.
+2. Add `model_control_mode=explicit` to config.properties file
```

Comment:
What's the priority b/w the two options? We can only offer this config.properties if it is too complicate for you to implement.

Reply:
Priority follows TorchServe standard as documented here https://github.com/pytorch/serve/blob/master/docs/configuration.md, will add a note in the docs that links to the priority